default/stringtables/en.txt

```diff
@@ -339,11 +339,13 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. You can specify the amount and repetition of the transfer. PP spent on the transfer are added to the imperial stockpile.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only transfers PP that are available at this planet so only PP produced in the same supply-group are used and no PP from the imperial stockpile.
 
-Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue.  This project is used to transfer PP to the stockpile even if there are things on the queue below it that could have been allocated available PP.'''
+Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue and may trigger a warning.  This project is used for fine-tuning allocation of PP to the stockpile even if there are items on the build queue below it that could have been allocated available PP.
```

Comment:
mentioning a warning seems unnecessary, and is awkwardly phrased / placed at the end of this sentence.

"for fine-tuning" -> "controlling"

comma after "stockpile" before "even"